common/reconciliation/invariant/openCurrentExecution.go

```diff
@@ -34,15 +35,17 @@ import (
 type (
 	openCurrentExecution struct {
 		pr persistence.Retryer
+		dc cache.DomainCache
 	}
 )
 
 // NewOpenCurrentExecution returns a new invariant for checking open current execution
 func NewOpenCurrentExecution(
-	pr persistence.Retryer,
+	pr persistence.Retryer, dc cache.DomainCache,
```

Comment:
I think you'll make the corresponding change in another PR? Do you mind adding a TODO here?

Reply:
but it has been used for fns []InvariantFactory

fns = append(fns, invariant.NewOpenCurrentExecution)